sirepo/template/cortex_sql_db.py

```diff
@@ -20,6 +20,32 @@ class Error(RuntimeError):
     pass
 
 
+def delete_material(material_id):
+    """Cascade delete all rows for a material"""
+
+    def _delete(session, table_name, field, value):
+        session.delete(table_name, PKDict({field: value}))
```

Comment:
add uid for security.

Reply:
This is tricky because the uid can't come from the client. And on the server it is running in a job_cmd, so there isn't a qcall to get the user info.

Reply:
simulation_db.user_path() pass to uid_from_dir_name()